src/main/core/manager.c

```diff
@@ -215,17 +215,39 @@ Manager* manager_new(Controller* controller, Options* options, SimulationTime en
     manager->hostsPath = g_build_filename(manager->dataPath, "hosts", NULL);
 
     if (g_file_test(manager->dataPath, G_FILE_TEST_EXISTS)) {
-        gboolean success = utility_removeAll(manager->dataPath);
-        utility_assert(success);
+        critical("data directory already exists");
+        return NULL;
     }
 
-    gchar* templateDataPath =
-        g_build_filename(manager->cwdPath, options_getDataTemplatePath(options), NULL);
-    if (g_file_test(templateDataPath, G_FILE_TEST_EXISTS)) {
-        gboolean success = utility_copyAll(templateDataPath, manager->dataPath);
-        utility_assert(success);
+    const gchar* templateDataPath = options_getDataTemplatePath(options);
```

Comment:
Might be worth breaking this out into a helper function. If so you can use early returns to unnest some of this logic. e.g.

```
if (!g_file_test(absTemplatePath, G_FILE_TEST_EXISTS) {
  critical("...")
  return
}
...
```
(Or if we upgrade these to `error` we can unnest either way)

Reply:
I made these return error, and now they're not longer nested.